src/rust/ide/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -98,6 +101,8 @@ ensogl::define_endpoints! {
         debug_pop_breadcrumb        (),
         /// Selects the breadcrumb by its index without sending signals to the controller.
         debug_select_breadcrumb     (usize),
+        /// Indicates the IDE is in text edit mode.
+        ide_text_edit_mode          (bool)
```

Comment:
This is about code editor focus? Please add more info to thr docs, so it will be obvious that it is a temporary solution when we will be refactoring focus management.